vault-admin-client/src/test/java/com/scality/osis/vaultadmin/impl/BaseTest.java

```diff
@@ -42,6 +47,13 @@
 
   protected static final String ENTITY_EXISTS_ERR = "The request was rejected because it attempted to create a resource that already exists.";
 
+  public static final String TEST_ACCESS_KEY = "access_key";
+  public static final String TEST_SECRET_KEY = "secret_key";
+  public static final String TEST_SESSION_TOKEN = "session_token";
+  public static final String TEST_SESSION_NAME = "session";
+  public static final String TEST_ROLE_ARN = "arn:aws:iam::0123456789:role/osis";
```

Comment:
An account ID is 12 digits.

Reply:
Done in 9101950281bb1e469ee5c3cbe333dcaa4c9a3953